var/spack/repos/builtin/packages/py-ipyvuetify/package.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyIpyvuetify(PythonPackage):
+    """
+    Jupyter widgets based on vuetify UI components which implement Google's
+    Material Design Spec with the Vue.js framework.
+    """
+
+    homepage = "https://github.com/widgetti/ipyvuetify/tree/master"
+    pypi = "ipyvuetify/ipyvuetify-1.9.0.tar.gz"
+
+    license("MIT")
+
+    maintainers("jeremyfix")
+
+    version("1.9.0", sha256="9c537e218299de32194b1da949d6b96bffe6c00f36bb6035409f2485feb881e7")
+
+    depends_on("python@3.6:", type=("build", "run"))
+    depends_on("py-setuptools@40.8.0:", type="build")
+    depends_on("py-wheel", type="build")
+
+    depends_on("py-ipyvue@1.7:2", type=("build", "run"))
+    depends_on("py-jupyter-packaging@0.7.10:", type=("build", "run"))
+    depends_on("py-jupyterlab@3.0.1:", type=("build", "run"))
+    depends_on("py-pynpm", type=("build", "run"))
```

Comment:
This doesn't seem to be a requirement

Reply:
I believe you refer to py-pynpm. But it is indeed a requirement I think.

In the pyproject.toml :
```
requires = [..., "pynpm"]
```

and then :

```
➜  ipyvuetify-1.9.0:  grep -nri pynpm
generate_source/generate_source.py:5:from pynpm import NPMPackage
grep: generate_source/__pycache__/generate_source.cpython-310.pyc: binary file matches setup.py:4:from pynpm import NPMPackage
pyproject.toml:2:requires = ["jupyter_packaging~=0.7.9", "jupyterlab~=3.0", "setuptools>=40.8.0", "wheel", "pynpm"]
```

Reply:
Ah sorry, for some reason I was looking at 1.8.10, not 1.9.0.